server.js

```diff
@@ -1,10 +1,442 @@
-const express = require("express");
+const express = require('express');
+
+const Product = require('./db.js');
+
 const app = express();
 const port = process.env.PORT || 8080;
 
-app.use(express.static('public'))
-app.use('/products/:productId', express.static('public'));
+app.use(express.static('public'));
+app.use(express.urlencoded());
+app.use(express.json());
+//app.use('/products/:productId', express.static('public'));
+
+app.get('/products/list', (req, res) => {
+  ///products/list?count=10&page=100
+  console.log('req.query', req.query);
+  const count = parseInt(req.query.count) || 5;
+  const page = parseInt(req.query.page) || 1;
+  Product.find({}, (err, docs) => {
+    if (err) {
+      console.log('Err finding docs', err);
+    }
```

Comment:
Does this code belong here? Best practice may be to have a controller file for your database which is what actually interacts with the database and you call those functions in your server. Separation of concerns.